mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -208,6 +209,8 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	FIELD field_41782 navigationType Lnet/minecraft/class_8015;
 	METHOD <init> (Lnet/minecraft/class_542;)V
 		ARG 1 args
+	METHOD method_0_2253 setWorld (Lnet/minecraft/class_638;Ljava/lang/String;)V
```

Comment:
I would like to say this mapping is kinda tentative since there isn't any direct same method in 1.20

Reply:
Tried to dig through yarn but I wasn't able to find any mappings for this method for 1.12 or 1.11. The method completely changes in (I assume) 1.13.